chatformatter-core/src/main/java/com/eternalcode/formatter/config/PluginConfig.java

```diff
@@ -44,31 +47,40 @@ public class PluginConfig implements ChatSettings, PlaceholderStack, TemplateRep
         "# RGB and gradient message usage: ",
         "# <color:#ff00ee>Example message</color>",
         "# <gradient:#ff00ee:#f79459>Example message</gradient>",
-        " "
+        "# ",
+        "# You can use three more placeholders: <displayname> <name> <message>",
```

Comment:
The plugin includes placeholders: 
<displayname> - in game nick
<name> -  ??? (please describle)
<message> - message sent by the player

Reply:
Name, displayname, message

Reply:
Displayname, name, message

Reply:
I mean, polish people will use our plugin and You know they need **special** explanation